runtime/jvmTest/resources/corner_cases/listing.txt

```diff
@@ -0,0 +1,18 @@
+number_1.0.json
```

Comment:
Do we have a 'golden value' for this tests or this is just a parser check, without correctness verification?

Reply:
We don't, unfortunately. I, of course, can hardcode it manually, but I am not sure this is a work that makes any sense

Reply:
What is quite possible is to test through roundtrips. In particular for valid values you can just serialize the result of deserialization and check that it is the same (and or equivalent - maybe ignore line breaks or something).